distribution/changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.2.0+ (in development)
 ------------------------------------------------------------------------
 - Feature: [#6998] Guests now wait for passing vehicles before crossing railway tracks.
+- Fix: [#7653] Finances money spinner is to narrow for big loans.
```

Comment:
Thanks for the patch!

Extremely minor, but please change 'to' to 'too' here.

Reply:
Ahah, my bad I copied it from the issue title - fixed 👍